src/main/java/io/reactivex/internal/observers/BasicFuseableObserver.java

```diff
@@ -28,13 +28,13 @@
 public abstract class BasicFuseableObserver<T, R> implements Observer<T>, QueueDisposable<R> {
 
     /** The downstream subscriber. */
-    protected final Observer<? super R> actual;
```

Comment:
fyi these are protected fields so will break folks. We'll try and work around this, just saying. cc @kojilin

Reply:
Internal.

https://github.com/ReactiveX/RxJava#ioreactivexinternal